src/Handler/CurlMultiHandler.php

```diff
@@ -95,17 +95,11 @@ public function tick()
         // Step through the task queue which may add additional requests.
         P\queue()->run();
 
-        if ($this->active &&
-            curl_multi_select($this->_mh, $this->selectTimeout) === -1
-        ) {
-            // Perform a usleep if a select returns -1.
-            // See: https://bugs.php.net/bug.php?id=61141
-            usleep(250);
+        $cms = curl_multi_select($this->_mh, $this->selectTimeout);
+        if ($cms != 0) {	
```

Comment:
Wrong comparison. Read docs: http://php.net/manual/en/function.curl-multi-select.php
```
On failure, this function will return -1
```
So, in this case you will do same action when it's failure and success. Also variable name is not clear. I think `$numberOfDescriptors` will be more cleaer. Finally it's better to write `$numberOfDescriptors >= 0`